### ISSUE TITLE
training procedure of base-lcm

### ISSUE THREAD

**Issue by guangyuli-uoe** [facebookresearch/large_concept_model#29]
hi,

really thanks for your impressive work !

just curious about if the training of base-lcm will contain all the steps below ?

def normalize(self, embeddings: Tensor) -> Tensor:
        if self.config.with_fft:
            embeddings = self.fft_transform(embeddings)

        embeddings = (embeddings - self.center) / self.scale
        if self.config.clip_proba is not None:
            embeddings = torch.clamp(embeddings, min=self.clip_min, max=self.clip_max)
        return embeddings
